plugins/woocommerce-admin/client/marketplace/utils/functions.tsx

```diff
@@ -13,26 +19,28 @@ interface ProductGroup {
 }
 
 // Fetch data for the discover page from the WooCommerce.com API
-const fetchDiscoverPageData = async (): Promise< Array< ProductGroup > > => {
-	const fetchUrl = MARKETPLACE_URL + '/wp-json/wccom-extensions/2.0/featured';
+async function fetchDiscoverPageData(): Promise< ProductGroup[] > {
+	let url = '/wc/v1/marketplace/featured';
```

Comment:
Interesting, I just realised that you can provide either a `path` or a `url`  to `apiFetch` depending on if you want to route locally or get an external URL.